containers/server-saline-image/Dockerfile

```
@@ -14,17 +14,17 @@ RUN groupadd -r --gid 10554 salt && \
 
 # Main packages
 RUN zypper ref -f && zypper --non-interactive up
-RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses --force-resolution --download-only saline python3-saline
+RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses --force-resolution --download-only saline python311-saline
 RUN find /var/cache/zypp/packages/ -type f -name '*.rpm'
 RUN rpm -ivh --nodeps $(find /var/cache/zypp/packages/ -type f \
     -name update-alternatives.rpm -o \
-    -name python3-base.rpm -o -name 'libpython3*.rpm' -o -name 'libopenssl1*.rpm' -o -name libexpat1.rpm -o \
-    -name salt.rpm -o -name python3-salt.rpm -o -name python3-tornado.rpm -o \
-    -name python3-contextvars.rpm -o -name python3-immutables.rpm -o -name python3-looseversion.rpm -o \
-    -name python3-packaging.rpm -o -name python3-distro.rpm -o \
-    -name python3-PyYAML.rpm -o -name 'libyaml*.rpm' -o -name python3-Jinja2.rpm -o -name python3-MarkupSafe.rpm -o \
-    -name python3-msgpack.rpm -o -name python3-more-itertools.rpm -o \
-    -name saline.rpm -o -name python3-saline.rpm -o -name python3-python-dateutil.rpm -o -name python3-six.rpm)
+    -name python311-base.rpm -o -name 'libpython311*.rpm' -o -name 'libopenssl1*.rpm' -o -name libexpat1.rpm -o \
```

I just tested it in my home branch and there is a problem with the name of libpython as it's called `libpython3_11`, and potentially could be a problem with `libopenssl` in future (right now it's not affected` but could be once any of the involved modules will be switched to `openssl3`, so maybe better to adjust the wildcard for `libopenssl` slightly in advance.
```suggestion
    -name python311-base.rpm -o -name 'libpython3_11*.rpm' -o -name 'libopenssl*.rpm' -o -name libexpat1.rpm -o \
```